BannerlordTwitch/BannerlordTwitch/Twitch/TwitchService.Bot.cs

```diff
@@ -19,6 +22,21 @@ namespace BannerlordTwitch
 {
     internal partial class TwitchService
     {
+        public dynamic GetClientFromClientId(Int64 userId)
+        {
+            WebRequest request = WebRequest.Create("https://api.twitch.tv/helix/users?id=" + userId);
```

Comment:
You should use the api instead, its `api.Helix.Users.GetUsersAsync(ids: new List<string>{ userId }, accessToken: authSettings.AccessToken).Result.User.FirstOrDefault()`